znflow/base.py

```
@@ -187,7 +187,9 @@ def __getitem__(self, item):
     def __add__(
         self, other: typing.Union[Connection, FunctionFuture, CombinedConnections]
     ) -> CombinedConnections:
-        if isinstance(other, (Connection, FunctionFuture, CombinedConnections)):
+        if isinstance(other, CombinedConnections):
+            return CombinedConnections(connections=[self] + other.connections)
```

Ruff (0.12.2)

Consider `[self, *other.connections]` instead of concatenation

Replace with `[self, *other.connections]`

(RUF005)